package.json

```diff
@@ -14,12 +14,12 @@
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
-    "@theforeman/builder": "^8.4.1",
-    "@theforeman/eslint-plugin-foreman": "^8.4.1",
-    "@theforeman/find-foreman": "^8.4.1",
-    "@theforeman/stories": "^8.4.1",
-    "@theforeman/test": "^8.4.1",
-    "@theforeman/vendor-dev": "^8.4.1",
+    "@theforeman/builder": "^8.9.0",
```

Comment:
this is not in packaging.
Should we use the packaged version (8.7.2)
I believe the others are just develop, but not sure if this wouldn't clash with the builder in packaging.